llvm/test/CodeGen/AMDGPU/llc-pipeline.ll

```diff
@@ -739,6 +739,8 @@
 ; GCN-O2-NEXT:    AMDGPU Remove Incompatible Functions
 ; GCN-O2-NEXT:    AMDGPU Printf lowering
 ; GCN-O2-NEXT:    Lower ctors and dtors for AMDGPU
+; GCN-O2-NEXT:    FunctionPass Manager
```

Comment:
Is it worth moving the pass slightly just to avoid instantiating another function pass manager here?